data/json/npcs/hints.json

```diff
@@ -120,7 +130,31 @@
     "Avoid using launchers in narrow hallways, you might miss.",
     "Met a mad chemist once.  She made a battery from a potato... or was it lemon?",
     "I have no idea why zombie grenadiers always holds a grenade in its hand.  Muscle memory?  Anyway, by no means you want to be near when it falls.",
-    "Is it dead yet?  How can you tell if it was dead before and it's dead now?  I prefer asking: are?  *smash* you?  *smash* dead?  *smash* yet?  *smash smash smash*",
-    "I hear there's strange big berries around, but why don't I hear first hand if they are filling for their size or not?"
+    "Is it dead yet?  How can you tell if it was dead before and it's dead now?  I prefer asking: are?  *smash*  you?  *smash*  dead?  *smash*  yet?  *smash smash smash*",
+    "I hear there's strange big berries around, but why don't I hear first hand if they are filling for their size or not?",
+    "I've heard of a gang called The Doctors. You know, bank robbers wearing stethoscopes. What are they trying to achieve? I use paper money to start fires now.",
+    "You can hole up on a roof if you need to.  Yeah, the rain will fall on your head, but it's harder for the dead to get you there. Get a tent, a rollmat, a sleeping bag and you're set.",
+    "I thought about putting a bag on my dog to carry some of my stuff.  It didn't work, because it was a chihuahua, and it was eaten by a rotweiler. Should have put some kevlar on it like those Z9. Oh well...",
```

Comment:
```suggestion
    "I thought about putting a bag on my dog to carry some of my stuff.  It didn't work, because it was a chihuahua, and it was eaten by a rottweiler.  Should have put some kevlar on it like those Z9.  Oh well...",
```